tests/Utilities/Plugin_Request_Utility_Tests.php

```diff
@@ -85,6 +85,9 @@ function( $checks ) {
 			}
 		);
 
+		$muplugins_loaded = $wp_actions['muplugins_loaded'];
+		unset( $wp_actions['muplugins_loaded'] );
```

Comment:
This feels like it should be in the `set_up` method.

Reply:
@spacedmonkey @felixarntz Felix's comment is correct, we're forcing the code to think it's initialized early. We also don't use this for all tests in this file so I don't think adding this to a `set_up` method would be the best solution here.

As we've progressed we've identified repeated code that we could refactor but I think it would be best to tackle this once the wider architecture is complete so we can tackle the refactoring as a whole.

Reply:
Okay, this is repeated code, it should be part of a trait.

Reply:
@spacedmonkey we've agreed that we're planning to address refactoring elements like this as part of the milestone review so that we don't refactor in isolation and  the wider team can contribute as part of this. As this is non-blocking I think this can wait until then.